server/src/main/java/org/apache/druid/indexing/overlord/IndexerMetadataStorageCoordinator.java

```diff
@@ -347,10 +347,13 @@ SegmentPublishResult commitReplaceSegments(
    * </ul>
    *
    * @param replaceSegments Segments being committed by a REPLACE task
+   * @param activeRealtimeSequencePrefixes Set of base sequence names of active and pending completion task groups
```

Comment:
Please update the javadoc to say prefixes instead of sequence names.

Reply:
Done